libs/domain/picking/src/models/picking-list.ts

```diff
@@ -2,6 +2,8 @@ import { PickingProduct } from './picking-product';
 
 export interface PickingList {
   id: string;
+  orderReference: string;
```

Comment:
We already have all `orderReferences` as an array, please use it instead of adding a single string.

Reply:
but we have something different in orderReferences 
<img width="346" alt="image" src="https://user-images.githubusercontent.com/43376934/235934024-b3bceb44-1473-4328-aaa0-f73b9180ae65.png">

Reply:
Ok if the order references are wrong you should fix those and not create a new one because it's basically the same thing so we don't need to duplicate `orderReferences` and `orderReference` props.